blazor/datagrid/events.md

```diff
@@ -2617,4 +2617,792 @@ The [OnBatchCancel](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grid
 }
 ```
 
+## FreezeLineMoving
+
+The [FreezeLineMoving](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridEvents-1.html#Syncfusion_Blazor_Grids_GridEvents_1_FreezeLineMoving) event is triggered when the freeze line is moving.
+
+> The freeze line can be enabled by setting the `AllowFreezeLineMoving` property to true.
```

Comment:
It is not like that freeze line dragging support is enabled while setting AllowFreezeLineMoving property. It is not correct that for enabling frozen line we need to set this property . change the description